src/isPalindrome.js

```diff
@@ -19,7 +19,10 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const word = str.replace(/[\W_]/g, '').toLowerCase();
+  const reversWord = word.split('').reverse().join('');
```

Comment:
what is `revers`? I do not know the word
```suggestion
  const reversedString = word.split('').reverse().join('');
```